packages/kbn-es/src/cli.ts

```diff
@@ -29,6 +29,9 @@ function help() {
 
       ${availableCommands.join('\n      ')}
 
+    To start serverless instance use '--projectType' flag as shortcut:
+      es --serverless=es
```

Comment:
Nit: I'd suggest to slightly reword it to make it a bit clearer, e.g. something like this:
```suggestion
    To start a serverless instance use the 'serverless' command with
    '--projectType' flag or use the '--serverless=<ProjectType>'
    shortcut, for example:

      es --serverless=es
```